src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine bulldozer = new Bulldozer();
+        Machine truck = new Truck();
+        Machine excavator = new Excavator();
 
+        bulldozer.doWork();
+        truck.doWork();
+        excavator.doWork();
+
+        bulldozer.stopWork();
+        truck.stopWork();
+        excavator.stopWork();
```

Comment:
let's create an array of `Machine` and call methods `doWork()` and `stopWork()` inside of loop